crates/task-impls/src/network.rs

```diff
@@ -292,6 +303,20 @@ impl<
                         TransmitType::Direct(membership.leader(vote.view_number() + 1)),
                     )
                 }
+                HotShotEvent::QuorumProposalRequestSend(view_number, sender_key) => (
+                    sender_key.clone(),
+                    MessageKind::<TYPES>::from_consensus_message(SequencingMessage::General(
+                        GeneralConsensusMessage::ProposalRequested(view_number, sender_key),
+                    )),
+                    TransmitType::Direct(membership.leader(view_number)),
+                ),
```

Comment:
This is a future change - but I believe a malicious node could put a `sender_key` that's not them and amplify traffic to them, as they would get the response proposal

Perhaps we could sign over this

Reply:
Yeah I know we had chatted about this before. Tobias and I had talked about it as well but the crux of the matter is that anyone at any time could spam a single node by adding a new internal event that always broadcasts and roasts the leader. I think we'll have to just penalize to avoid spamming, even if we sign it to ensure validity I can still get bogus data through the network layer to whomever I want, even if they end up dropping the message.

Reply:
Right, but I guess in this case you could essentially have all _other_ nodes DoS this one node, which might be harder to deal with

But long-term thing if any, not significant for this PR

Reply:
Actually previously the DataRequest message type was signed by the sender so this type of amplification/DDoS attack would not be possible, this PR is actually removing the signing.  We should added it back soon than later